src/index.html

```diff
@@ -1,15 +1,175 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="get"
+    >
+      <fieldset class="info-block">
+        <legend>Personal information</legend>
+        <div>
+          <label for="lname">Surname:</label>
+          <input
+            class="form-input"
+            type="text"
+            id="lname"
+            name="lname"
+            autocomplete="off">
+        </div>
+        <div>
+          <label for="fname">Name:</label>
+          <input
+            class="form-input"
+            type="text"
+            id="fname"
+            name="fname"
+            autocomplete="off">
+        </div>
+        <div>
+          <label for="age">How old are You?</label>
+          <input
+            class="form-input"
+            type="number"
+            id="age"
+            name="age"
+            autocomplete="off"
+            value="12"
+            min="12"
+            max="100">
+        </div>
+        <div>
+          <label for="birthdate">Full date of birth:</label>
+          <input
+            class="form-input"
+            type="date"
+            id="birthdate"
+            name="birthdate">
+        </div>
+        <div>
+          <label for="agree">I accept the term of the agreement</label>
+          <input
+            class="form-input"
+            type="checkbox"
+            id="agree"
+            name="agree">
```

Comment:
```suggestion
          <input
            class="form-input"
            type="checkbox"
            id="agree"
            name="agree"
          >
```
closing bracket on the new line as well